lib/nametag_db.ts

```diff
@@ -0,0 +1,48 @@
+import { NameTagContent } from '@/components/NameTagForm';
+import { getSession } from 'next-auth/react';
+
+export async function fetchNametagFromDB(): Promise<NameTagContent | undefined> {
+    const session = await getSession();
+
+    if (session && session.user) {
+        let nameTagToReturn = undefined;
+
+        await fetch("/api/auth/users/fetchUserData/nameTag?userEmail=" + session.user.email, { method: "GET" })
+        .then((res) => res.json())
+        .then((resJson) => {
+            if (resJson.success && resJson.nameTag) {
+                nameTagToReturn = resJson.nameTag;
+            }
+            else {
+                console.log("No nametag stored in DB.");
+            }
+        })
+        .catch((error) => {
+            console.error(error);
+        });
+
+        return nameTagToReturn;
+    }
+    else {
+        console.error("User session is not defined.");
+        return undefined;
+    }
+}
+
+export async function updateNameTagInDB(newNameTag: NameTagContent) {
+    const session = await getSession();
+
+    if (session && session.user) {
+        await fetch("/api/auth/users/updateUserData/nameTag", { 
```

Comment:
Do the `fetchUserData` and `updateUserData` need to have separate URLs, rather than just being discriminated by their method (i.e. having one be GET and the other POST)?

Reply:
Good idea, we could just combine them into one URL called `userData` and discriminate by GET vs POST.

Reply:
> Do the `fetchUserData` and `updateUserData` need to have separate URLs, rather than just being discriminated by their method (i.e. having one be GET and the other POST)?

Done: https://github.com/aimpowered/ZoomCompanion/pull/60/commits/88aa00b09e3296ec64b5b021e91211c6e9caab17